appinfo/update.php

```diff
@@ -1,18 +0,0 @@
-<?php
-/**
- * ownCloud - search_lucene
- *
- * This file is licensed under the Affero General Public License version 3 or
- * later. See the COPYING file.
- *
- * @author Jörn Friedrich Dreyer <jfd@butonic.de>
- * @copyright Jörn Friedrich Dreyer 2012-2015
- */
-
-$currentVersion = \OC::$server->getConfig()->getAppValue('search_lucene', 'installed_version');
-
-if (version_compare($currentVersion, '0.5.0', '<')) {
```

Comment:
Could also wrap with a call to:
```php
	/**
	 * Check if a table exists
	 *
	 * @param string $table table name without the prefix
	 * @return bool
	 * @since 8.0.0
	 */
	public function tableExists($table);
```